recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
That's a great question... I called this recipe ASL since it provides libasl, and because the [homebrew recipe](https://github.com/Homebrew/homebrew-science/blob/master/asl.rb) I was copying called it ASL. (I'm mainly interested in this as a dependency for IPOPT and other COIN solvers, which also refer to it as ASL.)

Maybe @tkelman or @tkralphs might be better able to answer on what this should be named?